apps/portal-api/src/migrations/20250805083254_pass_date_into_datetime.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export async function up(knex) {
+  // Pass subscriptions dates :
+  await knex.schema.alterTable('Subscription', (table) => {
+    table.dateTime('start_date_tmp');
+  });
+
+  await knex.raw(`
+    UPDATE "Subscription"
+    SET start_date_tmp = start_date
+  `);
+
+  await knex.schema.alterTable('Subscription', (table) => {
+    table.dropColumn('start_date');
+  });
+
+  await knex.schema.alterTable('Subscription', (table) => {
+    table.dateTime('start_date').nullable();
+  });
+
+  await knex.raw(`
+    UPDATE "Subscription"
+    SET start_date = start_date_tmp
+  `);
+
+  await knex.schema.alterTable('Subscription', (table) => {
+    table.dropColumn('start_date_tmp');
+    table.dropColumn('end_date');
+  });
+  // Update Service_Price
+  await knex.schema.alterTable('Subscription', (table) => {
+    table.dateTime('end_date').nullable();
+  });
+
+  await knex.schema.alterTable('Service_Price', (table) => {
+    table.dropColumn('start_date');
+  });
+  await knex.schema.alterTable('Service_Price', (table) => {
+    table.dateTime('start_date').nullable();
+  });
```

Comment:
I tried using raw SQL and you can directly change the column type to timestamp (I think equivalent to knex dateTime) by doing

```
ALTER TABLE "Subscription"
ALTER COLUMN start_date
TYPE timestamp;
```

So I think you can use:
```
await knex.schema.alterTable('Subscription', function (table) {
  table.dateTime('start_date').alter();
});
```